app.rb

```diff
@@ -146,6 +152,7 @@ def list_rental
   end
 
   def quit_app
+    Storage.write_people(@people)
```

Comment:
👍 Thanks for your effort on the method! 👍 However, it seems there's an issue with its functionality:

- [ ] **Storing Only People:** Currently, the method is storing only people when exiting the app. According to the project requirements, it should also store `books` and `rentals`. To address this, consider updating your code to ensure that both `books` and `rentals` are correctly stored when exiting the app.

By making this adjustment, you'll align the functionality with the project requirements. Keep up the good work! 🚀📚